CONTRIBUTING.md

```diff
@@ -7,9 +7,10 @@
 If you make changes to `lib/src/config.dart` and need to regenerate
 `lib/src/config.g.dart`:
 
-1. Uncomment the `json_serializable: ^3.0.0` dev dependency in `pubspec.yaml`
+1. Uncomment the `json_serializable` dev dependency in `pubspec.yaml`
```

Comment:
We keep it commented out because otherwise we hit the issue described below in this file:

> "This is necessary because a builder cannot depend on code that is generated by another builder in the same package. An alternative solution would be to define the `TestHtmlBuilderConfig` in a separate package, but this is simpler for now."